Source/OIDRegistrationRequest.h

```diff
@@ -95,7 +106,29 @@ NS_ASSUME_NONNULL_BEGIN
                  grantTypes:(nullable NSArray<NSString *> *)grantTypes
                 subjectType:(nullable NSString *)subjectType
     tokenEndpointAuthMethod:(nullable NSString *)tokenEndpointAuthMethod
-       additionalParameters:(nullable NSDictionary<NSString *, NSString *> *)additionalParameters
+       additionalParameters:(nullable NSDictionary<NSString *, NSString *> *)additionalParameters;
+
+/*! @brief Designated initializer.
+    @param configuration The service's configuration.
+    @param redirectURIs The redirect URIs to register for the client.
+    @param responseTypes The response types to register for the client.
+    @param grantTypes The grant types to register for the client.
+    @param subjectType The subject type to register for the client.
+    @param tokenEndpointAuthenticationMethod The token endpoint authentication method to register
+        for the client.
+    @param initialAccessToken The initial access token to access the Client Registration Endpoint
+        (if required by the OpenID Provider).
+    @param additionalParameters The client's additional registration request parameters.
+    @see https://openid.net/specs/openid-connect-registration-1_0.html#ClientRegistration
+ */
+- (instancetype)initWithConfiguration:(OIDServiceConfiguration *)configuration
+                         redirectURIs:(NSArray<NSURL *> *)redirectURIs
+                        responseTypes:(nullable NSArray<NSString *> *)responseTypes
+                           grantTypes:(nullable NSArray<NSString *> *)grantTypes
+                          subjectType:(nullable NSString *)subjectType
+              tokenEndpointAuthMethod:(nullable NSString *)tokenEndpointAuthenticationMethod
+                   initialAccessToken:(nullable NSString *)initialAccessToken
+                 additionalParameters:(nullable NSDictionary<NSString *, NSString *> *)additionalParameters
```

Comment:
line 131 is over 100 chars.

To resolve, the longest line of 125-131 (tokenEndpointAuthMethod) should be indented 4 spaces, and the rest lined up on that, as with the above initializer.